### ISSUE TITLE
Appoint Contributors

### ISSUE THREAD

**Issue by Swivelgames** [rimraf/k#97]
@supercrabtree People are posting issues, as well as Pull Requests, and many users still actively use this project. It might be time to appoint some contributors to the project so that it doesn't completely die.

Most projects would have been completely dead by now, but we are still posting PRs and Issues. I feel like this is a "now or never" type of situation. Pretty soon, people are going to stop using this. Which is really disappointing, because we all use and (attempt to) contribute to this because we really like what you've started and want to keep it going.

Thoughts?

**Comment by supercrabtree** [comment]
Yup, I completely agree. I appreciate the help you gave to bikegirl on the recent issue. If you are volunteering I am happy to add you as a contributor?

**Comment by reduxionist** [comment]
I'd also like to volunteer as a contributor, if desired. Thanks for k!
